test/controllers/GovSpendingControllerTest.scala

```diff
@@ -70,17 +70,17 @@ class GovSpendingControllerTest extends BaseSpec {
   "Calling Government Spend with no session"                                must {
     "return a 200 response" in {
       val controllerUnderTest = makeController(summaryJson)
-      val result2022          = Future.successful(controllerUnderTest.getAtsSaData("user", 2022)(request))
-      val result2023          = Future.successful(controllerUnderTest.getAtsSaData("user", 2023)(request))
+      val result2022          = Future.successful(controllerUnderTest.getAtsSaData("user", 2023)(request))
+      val result2023          = Future.successful(controllerUnderTest.getAtsSaData("user", 2024)(request))
 
       status(result2022.futureValue) mustBe 200
       status(result2023.futureValue) mustBe 200
     }
     "have the right data in the output Json" in {
 
       val controllerUnderTest = makeController(summaryJson)
-      val result2022          = Future.successful(controllerUnderTest.getAtsSaData(testUtr, 2022)(request))
-      val result2023          = Future.successful(controllerUnderTest.getAtsSaData(testUtr, 2023)(request))
+      val result2022          = Future.successful(controllerUnderTest.getAtsSaData(testUtr, 2023)(request))
+      val result2023          = Future.successful(controllerUnderTest.getAtsSaData(testUtr, 2024)(request))
```

Comment:
Is it too big a hassle to change these variable names? They had me confused why 2022 values were changing on an initial scroll through, maybe move from the year in the name if it's always the past 2 years

Reply:
Ah good point. I did change some, just missed these. I will change now

Reply:
Fixed